package.json

```diff
@@ -0,0 +1,54 @@
+{
+    "name": "zoid-ai-frontend",
+    "version": "0.1.0",
+    "private": true,
+    "devDependencies": {
+        "@testing-library/jest-dom": "^5.16.5",
+        "@testing-library/react": "^13.4.0",
+        "@testing-library/user-event": "^13.5.0",
+        "@types/jest": "^27.5.2",
+        "@types/node": "^16.11.58",
+        "@types/react": "^18.0.18",
+        "@types/react-dom": "^18.0.6",
+        "@typescript-eslint/eslint-plugin": "^5.36.2",
+        "@typescript-eslint/parser": "^5.36.2",
+        "eslint": "^8.23.0",
+        "eslint-config-prettier": "^8.5.0",
+        "eslint-plugin-react": "^7.31.7",
+        "husky": "^8.0.1",
+        "lint-staged": "^13.0.3",
+        "prettier": "2.7.1",
+        "react-scripts": "5.0.1",
+        "typescript": "^4.8.2"
+    },
+    "dependencies": {
+        "react": "^18.2.0",
+        "react-dom": "^18.2.0",
+        "web-vitals": "^2.1.4"
+    },
+    "scripts": {
+        "start": "react-scripts start",
+        "build": "react-scripts build",
+        "test": "react-scripts test --watchAll=false",
+        "eject": "react-scripts eject",
+        "precommit": "lint-staged --concurrent false"
+    },
+    "eslintConfig": {
```

Comment:
Can we move this section into the `.eslintrc.json` file?

Reply:
Have moved this to eslintrc.json file.

Tested it with trivial example:

```typescript
function App() {

    let num = 0;
    if (num) {
        const [state, setState] = useState<number>(0);
    }
   return (<div></div>);
}
```

This violates the rules of hooks as expected. Before moving eslintConfig section to .eslintrc.json, the error was not detected at all. Good catch!